packages/core/src/passport-adapter.ts

```diff
@@ -51,13 +51,18 @@ export function passportAuth(config: BlitzPassportConfig) {
       "No Passport strategies found! Please add at least one strategy.",
     )
 
-    const strategy = config.strategies.find((strategy) => strategy.name === req.query.auth[0])
-    assert(strategy, `A passport strategy was not found for: ${req.query.auth[0]}`)
+    const blitzStrategy = config.strategies.find(
+      (strategy) => strategy.strategy.name === req.query.auth[0],
```

Comment:
`strategy.strategy.name` looks a bit strange to me.

Think something like this would be a bit cleaner?
```suggestion
      ({strategy}) => strategy.name === req.query.auth[0],
```

Reply:
I've made the suggested change.